mcmd/commands/enable.py

```diff
@@ -72,3 +82,10 @@ def enable_theme(args):
     else:
         raise McmdError(
             'Applying theme failed. No themes found containing {} in the name'.format(args.theme, 'sys_set_StyleSheet'))
+
+
+@command
+def enable_language(args):
+    io.start('Enabling language {}'.format(highlight(args.language)))
+    url = urljoin(config.api('rest1'), 'sys_Language/{}/active'.format(args.language))
```

Comment:
should use the MOLGENIS Python REST API client

Reply:
See coments in #118